commands/Moderation/mute.js

```diff
@@ -0,0 +1,65 @@
+/*
+
+	To use this correctly, you will also need the unmute task located in
+	/tasks/unmute.js
+
+*/
+const { Command, Duration } = require('klasa');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			permissionLevel: 6,
+			requiredPermissions: ['MANAGE_ROLES'],
+			runIn: ['text'],
+
+			description: 'Mutes a mentioned user.',
+			usage: '<temp|run:default> [when:time|days:number] <member:user> [reason:string] [...]',
+			usageDelim: ' ',
+			subcommands: true
+		});
+	}
+
+	async run(msg, [, user, ...reason]) {
+		if (user.id === msg.author.id) throw 'Why would you mute yourself?';
+		if (user.id === this.client.user.id) throw 'Have I done something wrong?';
+
+		const member = msg.guild.members.get(user.id) || await msg.guild.members.fetch(user).catch(() => null);
```

Comment:
fetch already gets from cache if in cache

Reply:
This is due to already established conventions in ban and softban